src/common-model/functions.ts

```diff
@@ -134,39 +128,66 @@ export function getClinicalEntitySubmittedData(
 ): ClinicalInfo[] {
   const result = getClinicalObjectsFromDonor(donor, clinicalEntitySchemaName) as any[];
 
-  const clinicalRecords =
-    clinicalEntitySchemaName === ClinicalEntitySchemaNames.DONOR
-      ? result.map((entity: any) => ({
+  let clinicalRecords = [];
+
+  switch (clinicalEntitySchemaName) {
+    case ClinicalEntitySchemaNames.DONOR: {
+      clinicalRecords = result.map((entity: Donor) => ({
+        donor_id: donor.donorId,
+        program_id: donor.programId,
+        ...entity.clinicalInfo,
+      }));
+      break;
+    }
+    case ClinicalEntitySchemaNames.TREATMENT: {
+      clinicalRecords = result.map((treatment: Treatment) => {
+        const clinicalInfo = treatment.clinicalInfo || {};
+        const therapy_type =
+          treatment.therapies.length === 1
+            ? { therapy_type: treatment.therapies[0].therapyType }
+            : {};
+        const therapy_info =
+          treatment.therapies.length === 1 && treatment.therapies[0].clinicalInfo;
+        return {
           donor_id: donor.donorId,
           program_id: donor.programId,
-          ...entity.clinicalInfo,
-        }))
-      : clinicalEntitySchemaName === ClinicalEntitySchemaNames.TREATMENT
-      ? result.map((treatment: any) => {
-          const clinicalInfo = treatment.clinicalInfo || {};
-          const therapy_type =
-            treatment.therapies.length === 1
-              ? { therapy_type: treatment.therapies[0].therapyType }
-              : {};
-          const therapy_info =
-            treatment.therapies.length === 1 && treatment.therapies[0].clinicalInfo;
+          treatment_id: treatment.treatmentId,
+          ...clinicalInfo,
+          ...therapy_type,
+          ...therapy_info,
+        };
+      });
+      break;
+    }
+    case ClinicalEntitySchemaNames.SURGERY: {
+      clinicalRecords = result
+        .filter((treatment: Treatment) => {
+          const clinicalInfo: ClinicalInfo = treatment.clinicalInfo;
+          const treatmentType = <string[]>clinicalInfo['treatment_type'];
```

Comment:
for consistency the project as most type assertions in the form of `as` rather than angle brackets.
ie. `const treatmentType = clinicalInfo['treatment_type'] as string[];